single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/SvgParameters.java

```diff
@@ -0,0 +1,312 @@
+
+    public SvgParameters setCurrentValuePrecision(int currentValuePrecision) {
+        this.currentValuePrecision = currentValuePrecision;
+        return this;
+    }
+
+    public double getBusInfoMargin() {
+        return busInfoMargin;
+    }
+
+    public SvgParameters setBusInfoMargin(double busInfoMargin) {
+        this.busInfoMargin = busInfoMargin;
+        return this;
+    }
+
+    public double getFeederInfosIntraMargin() {
+        return feederInfosIntraMargin;
+    }
+
+    public SvgParameters setFeederInfosIntraMargin(double feederInfosIntraMargin) {
+        this.feederInfosIntraMargin = feederInfosIntraMargin;
+        return this;
+    }
+
+    public double getFeederInfosOuterMargin() {
+        return feederInfosOuterMargin;
+    }
+
+    public SvgParameters setFeederInfosOuterMargin(double feederInfosOuterMargin) {
+        this.feederInfosOuterMargin = feederInfosOuterMargin;
+        return this;
+    }
+
+    public boolean isFeederInfoSymmetry() {
+        return feederInfoSymmetry;
+    }
+
+    public SvgParameters setFeederInfoSymmetry(boolean feederInfoSymmetry) {
+        this.feederInfoSymmetry = feederInfoSymmetry;
+        return this;
+    }
+
+    public boolean isAddNodesInfos() {
+        return addNodesInfos;
+    }
+
+    public SvgParameters setAddNodesInfos(boolean addNodesInfos) {
+        this.addNodesInfos = addNodesInfos;
+        return this;
+    }
+
+    public boolean isUseName() {
+        return useName;
+    }
+
+    public SvgParameters setUseName(boolean useName) {
+        this.useName = useName;
+        return this;
+    }
+
+    public double getAngleLabelShift() {
+        return angleLabelShift;
+    }
+
+    public SvgParameters setAngleLabelShift(double angleLabelShift) {
+        this.angleLabelShift = angleLabelShift;
+        return this;
+    }
+
+    public boolean isLabelCentered() {
+        return labelCentered;
+    }
+
+    public SvgParameters setLabelCentered(boolean labelCentered) {
+        this.labelCentered = labelCentered;
+        return this;
+    }
+
+    public boolean isLabelDiagonal() {
+        return labelDiagonal;
+    }
+
+    public SvgParameters setLabelDiagonal(boolean labelDiagonal) {
+        this.labelDiagonal = labelDiagonal;
+        return this;
+    }
+
+    public boolean isTooltipEnabled() {
+        return tooltipEnabled;
+    }
+
+    public SvgParameters setTooltipEnabled(boolean tooltipEnabled) {
+        this.tooltipEnabled = tooltipEnabled;
+        return this;
+    }
+
+    public boolean isSvgWidthAndHeightAdded() {
+        return svgWidthAndHeightAdded;
+    }
+
+    public SvgParameters setSvgWidthAndHeightAdded(boolean svgWidthAndHeightAdded) {
+        this.svgWidthAndHeightAdded = svgWidthAndHeightAdded;
+        return this;
+    }
+
+    public CssLocation getCssLocation() {
+        return cssLocation;
+    }
+
+    public SvgParameters setCssLocation(CssLocation cssLocation) {
+        this.cssLocation = cssLocation;
```

Comment:
Also here for instance: null is an acceptable value?

Reply:
I added a non null check